packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -1415,7 +1515,16 @@ export class Combobox
 
   private renderChips(): JsxNode {
     const { activeChipIndex, readOnly, scale, selectionMode, messages } = this;
-    return this.selectedItems.map((item, i) => {
+
+    if (this.selectAllEnabled && this.allSelected) {
+      return null;
+    }
+
+    const filteredSelectedItems = this.selectedItems.filter(
+      (item) => item !== this.selectAllComboboxItemReferenceEl,
```

Comment:
fixme: Is the `selectAllComboboxItemReferenceEl` included in `combobox.selectedItems`? I'm not sure a user would expect that to be included in a public property since its not a "real" item.

Reply:
I think when we separated `this.keyboardNavItems`, we included `selectAllComboboxItemReferenceEl` in that, but removed it from `this.items`.
 
`getSelectedItems` is using `this.items`

Reply:
Ok so this isn't necessary then?